internal/network/command_network_link.go

```diff
@@ -0,0 +1,18 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func (c *command) newNetworkLinkCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "network-link",
```

Comment:
I think we already discussed this, but do you remember why we chose `confluent network-link` instead of `confluent network`?

Reply:
We may want to consider doing the latter. Recently we chose to introduce `confluent kafka consumer group` instead of `confluent kafka consumer-group`, so maybe we should follow the same pattern here.

Reply:
This decision predates my time but I think `confluent network link` should be fine, @kabraham13 thoughts on this naming?

Reply:
Old convo was here - https://confluentinc.atlassian.net/wiki/spaces/PM/pages/2987394300/1-pager+--+Network+CLI?focusedCommentId=3137868981

I am okay with `confluent network link ...` as it was my original choice

Reply:
Perfect, will update these commands. I assume this applies to `private-link` and `transit-gateway-attachment` as well?

Reply:
@ay-zhang we should keep those as is. `private-link` is a specific thing/construct so i think its okay. Same with `transit-gateway-attachment` its a very specific thing